OpenRA.Game/Map/MapPreview.cs

```diff
@@ -273,16 +273,16 @@ public MapPreview(Map map, ModData modData)
 
 			innerData = new InnerData
 			{
-				MapFormat = map.MapFormat,
+				MapFormat = ((IMap)map).MapFormat,
```

Comment:
unnessary cast? i assume map implements IMap.

Reply:
Map does not implement `IMap`, `DefaultMap` does. `Map` now only has the core functionality central to all maps, which is basically loading and checking the map, and getting its map format (see the new Map.cs for the remaining functionality of `Map`, which is now only 164 lines of code.